modules/openapi-generator-core/src/main/java/org/openapitools/codegen/config/GeneratorSettings.java

```diff
@@ -1198,7 +1198,6 @@ public Builder withHttpUserAgent(String httpUserAgent) {
          */
         public GeneratorSettings build() {
             GeneratorSettings instance = new GeneratorSettings(this);
-            //noinspection PlaceholderCountMatchesArgumentCount
```

Comment:
Obsolete since https://github.com/OpenAPITools/openapi-generator/pull/15553 fixed the reported problem.